internal/util/copy.go

```diff
@@ -83,6 +99,45 @@ func CopyIndexFiles(logger *log.CmdLogger, src, dest string) error {
 	return nil
 }
 
+// skipFolder checks if the folder will need to be skipped during indexes copy.
+func skipFolder(logger *log.CmdLogger, indexSrcDir, src string) (bool, error) {
+	path, _ := filepath.Rel(indexSrcDir, src)
+
+	matchContainsPublicIndexFolder, _ := filepath.Match("index*", path)
```

Comment:
The pattern should be more specific, i.e. `indexes/*`, otherwise things like "indexed/" would also match.